pym/gentoolkit/eclean/cli.py

```diff
@@ -63,10 +62,25 @@ def printUsage(_error=None, help=None):
         "merged-distfiles-options",
         "time",
         "size",
+        "invalid_paths",
     ):
         _error = None
     if not _error and not help:
         help = "all"
+    if _error == "invalid_paths":
+        print(
+            pp.error(
+                "eclean was not able to clean invalid binpkgs automatically due to older portage version"
```

Comment:
this can be taken misleadingly as there being an older portage version in the repository, stopping the clean.  please reword as `eclean was not able to remove invalid binpkgs due to missing features in the currently installed portage` or such

Reply:
Hey, Arsen sorry for the late response yes your message is more apt I'll use that.